casadm/cas_lib.c

```diff
@@ -1840,6 +1841,8 @@ int check_if_mounted(int cache_id, int core_id)
 			}
 			return FAILURE;
 		}
+		if (error)
+			return FAILURE;
```

Comment:
Don't we want to check `error` first? IMO we shouldn't assume that `difference` contains a meaningful value it `error` set to non-zero.

Reply:
good point